pkg/network/domainspec/generators.go

```diff
@@ -238,16 +108,34 @@ func (b *MacvtapLibvirtSpecGenerator) discoverDomainIfaceSpec() (*api.Interface,
 		mac = &podNicLink.Attrs().HardwareAddr
 	}
 
+	targetName, err := b.getTargetName(podNicLink)
```

Comment:
Now I see that you could have passed just `b.podInterfaceName` instead of the link.

Reply:
done.